go-controller/pkg/node/node_ip_handler_linux.go

```diff
@@ -97,6 +120,11 @@ func (c *addressManager) Run(stopChan <-chan struct{}) {
 						klog.Errorf("Failed to set node annotations: %v", err)
 					}
 				}
+			case <-initialSyncTimer.C:
```

Comment:
@trozet it is possible you meant for this to be a periodic timer?!? Maybe too paranoid? :)

Reply:
yeah please use a periodic timer, that is reset when we receive an event from addrChan